.github/workflows/actions/smoke-tests/action.yml

```diff
@@ -0,0 +1,34 @@
+name: Rspec Smoke Tests
+
+inputs:
+    environment:
+      description: "The name of the environment"
+      required: true
+
+jobs:
+  rspec:
+    name: Rspec
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Ruby 3.2.2
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2.2
+
+      - name: Bundle smoke test gems
+        shell: bash
+        run: |
+            gem install bundler
+            echo 'gem "rspec"' >> Gemfile
+            bundle
+
+      - name: Run ${{ inputs.environment }} tests
+        run: bin/bundle exec rspec --tag smoke_test -b
+        env:
+            SMOKE_TEST: "true"
+            CLAIMS_EXTERNAL_HOST: "track-and-pay-${{ inputs.environment }}.test.teacherservices.cloud"
```

Comment:
This is not valid for the production environment
Get the values from the yml file

Reply:
You can read it from the module output https://github.com/DFE-Digital/terraform-modules/blob/main/aks/application/outputs.tf#L6

Reply:
Or add logic in the workflow

Reply:
Fixed